project-base/storefront/hooks/cart/useOrderPagesAccess.ts

```diff
@@ -0,0 +1,36 @@
+import { useCurrentCart } from './useCurrentCart';
+import { getInternationalizedStaticUrls } from 'helpers/getInternationalizedStaticUrls';
+import { useDomainConfig } from 'hooks/useDomainConfig';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+
+export const useOrderPagesAccess = (page: 'transport-and-payment' | 'contact-information') => {
+    const router = useRouter();
+    const { cart, isFetching } = useCurrentCart();
+    const { url } = useDomainConfig();
+    const [canContentBeDisplayed, setCanContentBeDisplayed] = useState(false);
```

Comment:
What about naming which would follow the purpose of the hook like `isWithPageAccess`?

Reply:
See my comment above.